docs/fundamentals-and-concepts/stellar-data-structures/operations-and-transactions.mdx

```diff
@@ -148,53 +150,3 @@ Each operation must pass all the validity checks for an operation, described in
-
-### 4. Propagating (Validator)
-
-Once Stellar Core has determined that a transaction is valid, it will propagate the transaction to all other servers to which it’s connected. This way, a valid transaction is flooded to the entire Stellar network.
-
-### 5. Crafting a candidate transaction set (Validator)
-
-When it’s time to close the ledger, each Stellar Core validator takes all valid transactions it is aware of since the last ledger close and collects them into a candidate transaction set. If it hears about any incoming transactions now, it puts them aside for the next ledger close. If the number of operations in the candidate transaction set is greater than the maximum number of operations per ledger, transactions will be prioritized by their fee for inclusion in the set.
-
-### 6. Nominating a transaction set (Validator)
-
-Once each validator has crafted a candidate transaction set, the set is nominated to the network.
-
-### 7. Stellar Consensus Protocol (SCP) determines the final transaction set (Validator Network)
-
-SCP resolves any differences between candidate transaction sets and ultimately determines a single transaction set to apply, the close time of the ledger, and any upgrades to the protocol that need to be applied network-wide at the apply time.
-
-If a transaction doesn’t make it into the transaction set, it is kept around in memory to be added to the next transaction set on a best-effort basis.
-
-If a transaction is kept in memory after a certain number of ledger closes, it will be banned for several additional ledgers. This means no attempt will be made to include it in a candidate transaction set additional ledgers during this time.
-
-### 8. Transaction apply order is determined (Validator Network)
-
-Once SCP agrees on a particular transaction set, the apply order is computed for the transaction set. This shuffles the set's order to create uncertainty for competing transactions and maintains the order of sequence numbers for multiple transactions per account.
-
-### 9. Fees are collected (Validator)
-
-Fees are collected for all transactions simultaneously.
-
-### 10. Application (Validator)
-
-Each transaction is applied in the previously-determined order. For each transaction, the account’s sequence number is consumed (increased by 1), the transaction’s validity is rechecked, and each operation is applied in the order they occur in the transaction. Operations may fail at this stage due to errors that can occur outside of the transaction and operation validity checks. For example, an insufficient balance for a payment is not checked at submission and would fail at this time. The entire transaction will fail if any operation fails, and all previous operations will be rolled back.
-
-### 11. Protocol Upgrades (Validator)
```

Comment:
~~I hope all of this deleted stuff isn't going away forever?? It was super useful to have. Maybe in a encyclopedia entry?~~

Reply:
jk I see it later in the diff heh

Reply:
Yeah, pulled out the transaction lifecycle into its own section b/c we wanted to highlight it more. Will also add smart contract information in this section.